paper.md

```diff
@@ -1,36 +1,51 @@
 ---
-title: 'PyGBe: Python, GPUs and Boundary elements for biomolecular electrostatics'
+title: 'PyGBe: Python and GPU Boundary-integral solver for electrostatics'
 tags:
   - electrostatics
   - biophysics
   - Poisson-Boltzmann
```

Comment:
It's a new paper, associated to a new release of the software, with major enhancements (the LSPR solver). But the code repo still contains the whole PyGBe library.